### PR TITLE
[WOR-322] Remove references to cloud function ToS.

### PR DESCRIPTION
**Pull request by cahrens**
I ran firecloud locally and verified that I was prompted with the ToS when appropriate. There are automation tests, but apparently they are no longer run.

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback

- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [x] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/main/broadfcui/endpoints.cljs [hunk 1]
```diff
@@ -584,16 +584,3 @@
     :headers ajax/content-type=json
     :data (utils/->json-string tos-url)}
```
Comment (cahrens) [inline src/cljs/main/broadfcui/endpoints.cljs:585]: @MatthewBemis it looks like you added this in https://github.com/broadinstitute/firecloud-ui/pull/1607/files#diff-52b3931bbc7ac93073942c840c1ce432f92749138d26951a07b66a149db876fbR585

Why are you sending `tos-url`? It looks to me that `tos-url` is the cloud function URL (https://github.com/broadinstitute/firecloud-develop/pull/2919/files#diff-c20ec0c146ec8f9eb41e0e3590e6c98358574d92eef4a2a7de783eb0d7c972d8L32)

File: src/cljs/main/broadfcui/config.cljs [hunk 2]
```diff
@@ -65,6 +65,4 @@
 (defn terra-base-url [] (get @config "terraBaseUrl"))
 (defn terra-redirects-enabled [] (get @config "terraRedirectsEnabled" false))
 (defn tos-url [] (get @config "tosUrl"))
```
Comment (cahrens) [inline src/cljs/main/broadfcui/config.cljs]: I think I also want to delete `tos-url`, but have a question on the usage below (@MatthewBemis).

File: src/cljs/main/broadfcui/endpoints.cljs (hunk 1 above)
Comment (cahrens) [reply src/cljs/main/broadfcui/endpoints.cljs:585]: Or maybe `tos-url` just happens to be the same name as what is defined in https://github.com/broadinstitute/firecloud-ui/pull/1610/files#diff-8800b72bac2b6918985e4cb588d743bbad79f91e40491a4eb9c4163ea35260ceL67? I'm unfamiliar with `cljs`, so it's hard for me to know.

Comment (MatthewBemis) [reply src/cljs/main/broadfcui/endpoints.cljs:585]: The `tos-url` being passed into this is coming from here: https://github.com/broadinstitute/firecloud-ui/pull/1607/files#diff-38d6a2a825de69453b37819e66002057e11717a3d981ae55cc06faa712d3792bR334

So yes, your 2nd comment is correct

Comment (cahrens) [reply src/cljs/main/broadfcui/endpoints.cljs:585]: Thanks!

**Review by aherbst-broad — APPROVED** [review]
lgtm

**Review by marctalbott — APPROVED** [review]

**Pull request merged**